src/modules/fancyzones/editor/FancyZonesEditor/Properties/Resources.resx

```diff
@@ -145,7 +145,7 @@
     <value>Name</value>
   </data>
   <data name="Note_Custom_Table" xml:space="preserve">
-    <value>Note: Hold down Shift Key to change orientation of splitter</value>
+    <value>Note: Hold down Shift Key to change orientation of splitter.  To merge zones, select the zones and click "merge".</value>

```

Comment:
It may be worthwhile to include 'click and drag' in this information, as select usually indicates 'click each', but in this case will just add more dividers.

Not a wordsmith, but something along these lines:
"To merge zones, hold down shift and click and drag across desired zones, then release mouse button and click 'merge".